track.go

```diff
@@ -235,6 +271,7 @@ func (track *baseTrack) bind(ctx webrtc.TrackLocalContext, specializedTrack Trac
 					track.onError(err)
 					return
 				}
+				track.onRTPWrite(pkt)
```

Comment:
do we care about errors like we do with onImageRead ?

Reply:
no, I don't think we should, because an error will be handled above and never write a packet